Protection Profile/BiocPP.adoc

```diff
@@ -455,37 +457,11 @@ Target error rates defined in SFR shall be evaluated based on <<BIOSD>>. Normall
 
 ===== FIA_PAD_EXT.1 Presentation attack support [[FIA_PAD_EXT.1]]
 
-*FIA_PAD_EXT.1.1*:: The TSF shall provide the [*selection*: _biometric modality_] with [*selection*: _enrolment detection_] detection at biometric enrolment and [*selection*: _verification detection equal to or greater than enrolment detection_] detection at biometric verification.
+*FIA_PAD_EXT.1.1*:: The TSF shall provide the [*selection*: _eye, face, fingerprint, vein_] with [*selection*: _PAD-L3, PAD-L2, PAD-L1, no PAD_] detection as specified in FIA_MBE_EXT.3 during biometric enrolment and [*selection, equal or greater than enrolment*: _PAD-L3, PAD-L2, PAD-L1, no PAD_] detection as specified in FIA_MBV_EXT.3 during biometric verification.
 
-*Application Note {counter:remark_count}*:: The <<PAD_Levels>> table provides acceptable options for each modality by row for each selection within the SFR. Selecting a specific modality then specifies the options that are available for enrolment and verification detection.
+*Application Note {counter:remark_count}*:: Not all combinations of modality and PAD levels may be supported.
```

Comment:
This should be updated to say that the website lists the available combinations of PAD and modality that are supported.